### DISCUSSION TITLE
Heroku doesn't recognize GitHub student account

### DISCUSSION THREAD

**Discussion by sspahicparagon** [community/community#39703] [GitHub Education] [answered]
Hello everyone. Not sure if this is the right place to ask, I saw that maybe 2 or 3 topics were aligned for this issue.

So, maybe 2 or 3 hours ago, I applied for the GitHub student program. It seems that I was authorized, since when I got to the page that requests my data, it says that I'm already a member as of November 21, 2022.

Now that's great! But why I primarily wanted to join is because of Heroku. The issue is that when I go to the designated Heroku page for the Student package, I get the following screen.

![image](https://user-images.githubusercontent.com/80258070/203181863-786180f0-aea2-4eee-806f-37c87c091b00.png)

Does it mean I should wait? I went ahead and deployed an app of mine already. Heroku it seems is dropping the free package on November 28, 2022. So I'm a bit cutting it close.

Any advice?

**Comment by samus-aran** [comment]
Hi @sspahicparagon 👋 How did you get on? Did you re-try via Heroku to verify your Global Campus status?

**Reply by sspahicparagon** [reply]
Hello @samus-aran . I went through GitHub and sent a request.
![image](https://user-images.githubusercontent.com/80258070/204087887-54da44b1-7f9f-4047-9f69-f20786f30dec.png)
But when I click to verify my account GitHub account on Heroku, I get redirected back to the home page. I checked, I'm using the same email that I used for GitHub.

![image](https://user-images.githubusercontent.com/80258070/204088394-9ce660c4-393e-43f1-9884-7474ea887ed7.png)

**Reply by prasadpatil10002** [reply]
@sspahicparagon  any solution?

**Comment by barni363hun** [comment, marked as answer]
Hey! Today I faced the same problem.
You can apply for the Heroku for GitHub Students Offer In an 🌒**incognito window**🌒!

**Reply by sspahicparagon** [reply]
@barni363hun  Man o man... That's it. Great job, thank you!

**Reply by MadeleineWalder** [reply]
This doesn't work for me?
How exactly do you do this when you are required to login anyway? 
Iv'e been trying to get this to work for 3 days now :((

**Reply by designlandwebsolutions** [reply]
It doesn't work with me too, can any one help

**Reply by MadeleineWalder** [reply]
@designlandwebsolutions this is what i did to get Heroku working maybe it will work for you:
1. Logout of ALL instances of Heroku and Github
2. Restart your PC 
3. Clear browser cookies and cache
4. Login to Github
5. Go to [Github Education](https://education.github.com/pack/offers) and try to activate the Heroku student offer again

**Reply by BilalShah3** [reply]
incognito window method work for me

**Reply by BilalShah3** [reply]
you need to logout from git hub and heroku and then sign in again in incognito window

**Comment by BrownCAB** [comment]
Does anyone know when the credits will show up?

**Reply by sspahicparagon** [reply]
Hello @BrownCAB ,
The credits showed up for me when I bought a package.

**Reply by BilalShah3** [reply]
12 to 24 hours when you applied successfully

**Comment by samus-aran** [comment]
Hi @designlandwebsolutions & @BrownCAB & @MadeleineWalder  👋 

We first request that anyone having issues with redeeming their Heroku benefits to raise a [Heroku helpdesk ticket](http://help.heroku.com/). Let us know how you get on 👍

**Comment by jmllubes** [comment]
Hi I'm unable to verify heroku account with github, I tried a lot of things but It keeps saying. I created a ticket in heroku support but they say that is github problem...
<img width="845" alt="Captura de pantalla 2023-02-21 134820" src="https://user-images.githubusercontent.com/16512469/220389849-11159d8d-b6a7-4870-b7ee-54e586a6891c.png">

**Reply by BilalShah3** [reply]
make your request on incognito window

**Comment by MadeleineWalder** [comment]
I had the exact same problem and it took ages to get any help from anyone.
When I finally did this is what they told me:

1. Log out of github/gitpod and heroku completely
2. Clear all your cookies and cache the browser
3. Restart your pc

This worked for me, I hope it helps!

**Reply by jmllubes** [reply]
It doesn't work for me. I tried with diferent computers, browsers, incognito modes....

**Reply by BilalShah3** [reply]
you nee to logout from git hub and heroku and then sign in again in incognito window

**Comment by JustLearningMX** [comment]
Hi,  I am from Mexico and I'm trying to join to Heroku with my github student pack account but it's not working, even cleaning cookies, incognito modes, etc. The link "Get access by connecting your GitHub account on Heroku" only shows a page with information but there is not an option to connect with Heroku. I previously had an account with Heroku but I need to use the benefit for my scholars projects. Thanks for help me!

**Comment by Markyjay** [comment]
I have also run into this problem. I have followed each suggestion above, including using incognito, clearing cache, logging in and out of sites and restarting, multiple times, I used my credit card to gain access to Heroku but the credits for the student program are not showing, I have also spoken to a person from the helpdesk at code institute who guided me through the same processes as mentioned and I have also sent a ticket to Heroku who responded by saying that the problem is on the GitHub side and that I should contact them and they also sent me here to this discussion. I can't believe I need to buy a package to get the credits. Its the only option I have not tried can anyone tell me if I have any other option? Or if buying a random package is sure to work? Can I get my money back if package not needed?  Thank you for any help, much appreciated.

**Reply by BilalShah3** [reply]
I dont clear my cache either just follow below steps:
 logout from git hub and heroku
 sign in again in incognito window on heroku and git hub
note make request on incognito window go to your git student account and from there you see a list of free services select heroku and make request

**Comment by MuhamedSaad112** [comment]
I have also run into this problem. I have followed each suggestion above, including using incognito, clearing cache, logging in and out of sites and restarting, multiple times, I used my credit card to gain access to Heroku but the credits for the student program are not showing, I have also spoken to a person from the helpdesk at code institute who guided me through the same processes as mentioned and I have also sent a ticket to Heroku who responded by saying that the problem is on the GitHub side and that I should contact them and they also sent me here to this discussion. I can't believe I need to buy a package to get the credits. Its the only option I have not tried can anyone tell me if I have any other option? Or if buying a random package is sure to work? Can I get my money back if package not needed? Thank you for any help, much appreciated.
